src/utils.js

```diff
@@ -466,7 +466,7 @@ exports.insertPixel = function (url) {
     } catch (e) {
     }
   };
-  this.insertElement(img);
+  exports.insertElement(img);
```

Comment:
should this reference be updated too? https://github.com/prebid/Prebid.js/blob/35e68ae0d5deb648fc9178b9730362f03d10de5e/src/utils.js#L482
(no pun intended)